examples/java-jsp/README.md

```diff
@@ -0,0 +1,152 @@
+# Java JSP Example
+
+This project demonstrates use of verifiable credentials and presentation to enable
+user authentication for an application.
+
+## Dependencies
+
+### Java
+
+To run this example you will need Java installed.
+
+For example, on Ubuntu you could install OpenJDK:
+
+```bash
+# apt-get install openjdk-11-{jre,jdk}
+```
+
+### Maven
+
+To install Maven, run the following command:
+
+```bash
+# apt-get install maven
+```
+
+### Tomcat
+
+To install tomcat locally, you could run the following sequence of commands,
+sourced from
+[Digital Ocean's Tutorial](https://www.digitalocean.com/community/tutorials/install-tomcat-9-ubuntu-1804)
+
+Create `tomcat` user and group.
+```bash
+# groupadd tomcat
+# useradd -s /bin/false -g tomcat -d /opt/tomcat tomcat
+```
+
+Download and extract `tomcat` to `/opt/tomcat`.
+```bash
+$ curl -O https://downloads.apache.org/tomcat/tomcat-9/v9.0.43/bin/apache-tomcat-9.0.43.tar.gz
+# mkdir /opt/tomcat
+# tar xzvf apache-tomcat-*tar.gz -C /opt/tomcat --strip-components=1
+```
+
+Set permissions and groups.
+```bash
+# chgrp -R tomcat /opt/tomcat
+# chmod -R g+r /opt/conf
```

Comment:
should the paths in this section be prefixed with `/opt/tomcat` as in `/opt/tomcat/conf` instead of `/opt/conf`?